common/scala/src/main/scala/org/apache/openwhisk/common/Logging.scala

```diff
@@ -593,7 +593,8 @@ object LoggingMarkers {
   val SCHEDULER_KAFKA = LogMarkerToken(scheduler, kafka, start)(MeasurementUnit.time.milliseconds)
   val SCHEDULER_KAFKA_WAIT_TIME =
     LogMarkerToken(scheduler, "kafkaWaitTime", counter)(MeasurementUnit.none)
-  def SCHEDULER_WAIT_TIME(action: String) = LogMarkerToken(scheduler, "waitTime", counter, Some(action), Map("action" -> action))(MeasurementUnit.none)
+  def SCHEDULER_WAIT_TIME(action: String) =
+    LogMarkerToken(scheduler, "waitTime", counter, Some(action), Map("action" -> action))(MeasurementUnit.none)
```

Comment:
Seems the content is same, just do format.
Normally, the time unit should be `MeasurementUnit.time.milliseconds`, why here `MeasurementUnit.none`?

Reply:
I just kept the original one.
I would update it.